src/approaches/pg3_approach.py

```diff
@@ -324,26 +342,25 @@ def _get_atom_plan_for_task(self, ldl: LiftedDecisionList,
 
     @staticmethod
     def _count_missed_steps(ldl: LiftedDecisionList,
-                            atom_seq: Sequence[Set[GroundAtom]],
+                            atoms_seq: Sequence[Set[GroundAtom]],
                             goal: Set[GroundAtom]) -> float:
         missed_steps = 0.0
-        for t in range(len(atom_seq) - 1):
-            ground_nsrt = utils.query_ldl(ldl, atom_seq[t], goal)
+        for t in range(len(atoms_seq) - 1):
```

Comment:
i saw some stray uses of `atom_seq` in `test_datasets.py` as well, want to just lump that in here?

Reply:
(or a new PR or just leave an issue for now)

Reply:
#1020